docs/source/backends/spark.rst

```diff
@@ -42,11 +42,11 @@ Use ``ibis.spark.connect`` to create a client.
 
 PySpark client (Experimental)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-.. currentmodule:: ibis.pyspark.api
+.. currentmodule:: ibis.backends.pyspark
 
-The PySpark client is accessible through the ``ibis.pyspark`` namespace.
+The PySpark client is accessible through the ``ibis.backends.pyspark`` namespace.
 
-Use ``ibis.pyspark.connect`` to create a client.
+Use ``ibis.backends.pyspark.connect`` to create a client.
```

Comment:
Same for this one, you can leave it as it was.

Reply:
@datapythonista I have reverted the changes made to spark.rst file and committed it. Saw your comment below. Should I have follow the steps from git fetch onwards and then push the changes?

Reply:
Yes, that would be great. Copy your ibis directory first, in case things go wrong.

Reply:
yup took the backup.

Reply:
@datapythonista pushed the changes. Do I need to do anything else?

Reply:
Yes, please open a new PR.